.github/workflows/build-sphinx.yml

```diff
@@ -41,12 +41,11 @@ jobs:
           python-version: 3.x
       - name: Install dependencies
         run: pip install numpy cython sphinx sphinx_rtd_theme
-      - name: set Intel OneAPI environment
-        run: . /opt/intel/oneapi/setvars.sh
-      - name: Build library
-        run: ./0.build.sh
       - name: Build docs
-        run: ./0.builddoc.sh
+        run: |
+         . /opt/intel/oneapi/setvars.sh
+         ./0.build.sh
```

Comment:
`./0.build.sh: No such file or directory` because `working-directory: doc` for step `Build docs`.

Reply:
https://github.com/IntelPython/dpnp/runs/1506862902?check_suite_focus=true